apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-docs.component.html

```diff
@@ -0,0 +1,49 @@
+<fd-docs-section-title id="formDatePicker" componentName="DatePicker">
+    Simple Datepicker and Range Datepicker in Platform Form.
```

Comment:
Change "Datepicker" to "Date Picker", it should be two words.

Reply:
fixed